.goreleaser.yaml

```diff
@@ -110,7 +110,7 @@ checksum:
   name_template: checksums.txt
 
 snapshot:
-  name_template: "{{ incpatch .Version }}-snapshot"
+  name_template: "{{ incminor .Version }}-snapshot"
```

Comment:
What about changing this to `-nightly` instead of `-snapshot`? Checked two projects, Goreleaser and Rust, they both use `-nightly`. It might seem more natural to users